instrumentation/spring/spring-web/spring-web-3.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/springweb/v3_1/SpringWebInstrumentationModule.java

```diff
@@ -23,7 +24,9 @@ public SpringWebInstrumentationModule() {
   @Override
   public ElementMatcher.Junction<ClassLoader> classLoaderMatcher() {
     // class added in 3.1
-    return hasClassesNamed("org.springframework.web.method.HandlerMethod");
+    return hasClassesNamed("org.springframework.web.method.HandlerMethod")
+        // class added in 6.0
+        .and(not(hasClassesNamed("org.springframework.web.ErrorResponse")));
```

Comment:
should we wait on this change until we introduce the spring 6 instrumentation?

Reply:
Nope, I think this should be merged before that - the `spring-web` instrumentation is supposed to work in tandem with `spring-webmvc`; without `spring-webmvc` `spring-web` will just try to load a class that does not exist and silently fail.